src/modules/pos/validator/components/CommissionHistory/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { useCurrentAccount } from '@account/hooks';
+import { useTransactionEvents } from '@transaction/hooks/queries';
+import Dialog from '@theme/dialog/dialog';
+import Box from '@theme/box';
+import BoxHeader from '@theme/box/header';
+import BoxContent from '@theme/box/content';
+import { QueryTable } from '@theme/QueryTable';
+import header from './header';
+import CommissionHistoryRow from './CommissionHistoryRow';
+import styles from './styles.css';
+
+const CommissionHistory = () => {
+  const { t } = useTranslation();
+  const [currentAccount] = useCurrentAccount();
+  return (
+    <Dialog hasClose>
```

Comment:
Modal does not seem to be scrollable. Test having made many commission changes, and then the first ones 100 -> 99 are hidden and cannot scroll to see them

Reply:
Fixed